Sum-of-two-lowest-positive-integers/getSum.js

```diff
@@ -0,0 +1,13 @@
+const getSum = (a, b) => {
```

Comment:
according to readme this function should take an array

Reply:
wrong file